Assets/Scripts/GameAudio/AmbientAudioController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using GameTime;
+using Managers;
+using UnityEngine;
+
+namespace GameAudio
+{
+    public class AmbientAudioController : MonoBehaviour
+    {
+        [SerializeField][Tooltip("Clock which is managing the game time ")]
+        private Clock clock;
+
+        private void OnEnable()
+        {
+            Clock.SunRise += DayAmbience;
+            Clock.SunSet += NightAmbience;
+            Clock.MoonRise += MoonRiseAmbience;
+            Clock.MoonSet += MoonShineAmbienceStop;
+        }
+
+        private void OnDisable()
+        {
+            Clock.SunRise -= DayAmbience;
+            Clock.SunSet -= NightAmbience;
+            Clock.MoonRise -= MoonRiseAmbience;
+            Clock.MoonSet -= MoonShineAmbienceStop;
+        }
+
+        private void Start()
+        {
+            AudioManager.Instance.FadeIn(clock.IsNight ? "AmbCrickets" : "AmbBirds", 5);
```

Comment:
Could you add the sound names as constants or as serialised parameters? Makes it less hard-coded :)

Reply:
done